docker-compose.yaml

```diff
@@ -73,12 +111,17 @@ services:
             USER_ROOT_USERNAME: ${ADMIN_USER}
             USER_ROOT_PASSWORD: ${ADMIN_PASS}
             USER_ROOT_EMAIL: ${ADMIN_EMAIL}
-            SUPERVISOR_HOST: ${SUPERVISOR_HOST}
-            SUPERVISOR_PORT: ${SUPERVISOR_PORT}
+            BACKEND_SUPERVISOR_HOST: ${BACKEND_SUPERVISOR_HOST}
+            BACKEND_SUPERVISOR_PORT: ${BACKEND_SUPERVISOR_PORT}
+            MON_SUPERVISOR_HOST: ${MON_SUPERVISOR_HOST}
+            MON_SUPERVISOR_PORT: ${MON_SUPERVISOR_PORT}
             JWT_SECRET_KEY: ${JWT_SECRET_KEY}
             FLASK_SECRET_KEY: ${FLASK_SECRET_KEY}
             JS_VERSION: ${JS_VERSION}
             SECURITY_PASSWORD_SALT: ${SECURITY_PASSWORD_SALT}
+            # for backwards compatibility
```

Comment:
I do not understand why we need this. In which case scenario SUPERVISOR_HOST, SUPERVISOR_PORT can exist while BACKEND_SUPERVISOR_HOST, etc..  is used?

Reply:
in case people forget to do `docker-compose pull`. this is a safety lock till the next release and then we can ditch it.